spec/fixtures/release/find_2225dd4c-ae9a-403b-8ea0-9e05014c778f.xml

```diff
@@ -37,6 +37,7 @@
     <medium-list count="1">
       <medium>
         <position>1</position>
+        <format id="9712d52a-4509-3d4b-a1a2-67c88c643e31">CD</format>
```

Comment:
I think I re-recorded these not so long ago, was the `format` added recently?

Reply:
No, this specific release is still without a format. I added it manually just to be able to test against it.

If consistency with the prod data is important to you, and if you know this release to be a CD, I guess you could edit it on Musicbrainz directly? 😛

Reply:
Alright, cool. Just thought I was missing something.